templates/map-canvas.html

```diff
@@ -34,10 +34,19 @@
         <div id="inputHolder" style="display: flex">
             <input type="text" id="mapCanvasZoomLevel" value="" placeholder="z" style="align-self: flex-start" disabled>
             <input type="text" id="mapCanvasSearchBox" name="placesSearch" placeholder="Search">
+            <input type="text" id="mapCanvasSceneName" name="sceneName" placeholder="SceneName">
             <input type="button" onClick="updateScene()" value="Update Scene">
             <input type="button" onClick="generateScene()" value="Generate Scene">
+            <input type="button"  id ="generate-surrounding" value = "Generate Expanded Scene">
```

Comment:
Nit pick: Spacing / white space.

Reply:
No I made some further changes, can you have a look and let me know if you can merge them?